libs/libcommon/include/common/avx2_mem_utils.h

```diff
@@ -150,42 +155,64 @@ FLATTEN_INLINE_PURE static inline bool check_block32x4_eq(const char * a, const
 
 FLATTEN_INLINE_PURE static inline int cmp_block32x4(const char * a, const char * b)
 {
-    if (check_block32x4_eq(a, b))
+    if (likely(check_block32x4_eq(a, b)))
```

Comment:
why add `likely` here?

Reply:
To make sure compiler will put such branch closer.

Reply:
By now, compiler choose the expected way.